src/components/new-request/name-check/name-check.vue

```diff
@@ -220,6 +215,7 @@ import { baseItemsConflicts, baseItemsStructure } from './resources'
 })
 export default class NameCheck extends Vue {
   @Getter getConflictsConditional!: Array<string>
+  @Getter getConflictsConditionalInstructions!: Array<{ word: string, instructions: string }>
```

Comment:
Although a simple interface, it could be made into it's own in the event you need to apply type or a return type based on this getter :)